### ISSUE TITLE
Can we print out the Error message?

### ISSUE THREAD

**Issue by AKwoKWH** [vra/bing_brush#3]
I am getting the below message, but unable to see what is wrong.

==> Generating...
==> Your boosts have run out, using the slow generating pipeline, please wait...
==> Error occurs, please submit an issue at https://github.com/vra/bing_brush, I will fix it as soon as possible.

**Comment by AllFatherAI** [comment]
Hey @AKwoKWH I've created a PR with the fix. Please give it a try: https://github.com/vra/bing_brush/pull/4/files#diff-e6759c5cd62a87bc418d56272a90976a8d9742f836fe81ef3707eda597eec462R60

**Comment by AKwoKWH** [comment]
Prefect, it works!
But I have a question, why you need to override the IP address to a random one? If the user have a US IP already, he doesn't need this random ip settings, can simply use his own static IP so making the request more "trustworthy" to Bing.

**Comment by AllFatherAI** [comment]
@AKwoKWH Yes, you are correct. If you are from the USA and you are using only one account for yourself, you can leave your IP.

**Comment by AKwoKWH** [comment]
@AllFatherAI 
How about having an option to this?
A parameter to set whether using a random ip or current ip.
